spec/grape/api_spec.rb

```diff
@@ -3209,6 +3209,41 @@ def static
           expect { a.mount b }.to_not raise_error
         end
       end
+
+      it 'should correctly include module in nested mount' do
+        module Test
```

Comment:
This may pollute global scope. Either call the module something like `ApiSpecTest` or define it dynamically.